### PR TITLE
Added html for section 1

### PR DESCRIPTION
**Pull request by NickOne19**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,13 +4,31 @@
...
+          <a class = "link" href="https://google.com">
+            <img src="./assets/icons/google.svg" alt="Google icon">
+          </a>
+        </div>
+        <div class="image-block">
+          <img src="./assets/images/1.jpg" alt="Bag and device">
+        </div>
+      </div>
+    </div>
+  </section>
```
Comment (nagorapro) [inline index.html:31]: Use BEM methodology
Use "download" class name
Use "download" id name

**Review by nagorapro — COMMENTED** [review]
Need fix

**Review by NickOne19 — COMMENTED** [review]
Need fix

**Review by NickOne19 — COMMENTED** [review]
Fixed

File: index.html [hunk 2]
```diff
@@ -4,13 +4,31 @@
...
+  <section class="download" id ="download">
+    <div class="download__wrapper">
+      <div class="download__text-block">
+        <h1 class="download__title">Peaceful shopping mindful money</h1>
+        <p class="download__copy">See how we can help with making your shopping experience and money management more ZEN.</p>
+        <div class="download__links">
+          <a class = "download__link" href="https://apple.com">
+            <img src="./assets/icons/apple.svg" alt="Apple icon">
+          </a>
+          <a class = "download__link" href="https://google.com">
```
Comment (nagorapro) [inline index.html:22]:
````
<a class="download__link" href="https://google.com">

**Review by nagorapro — COMMENTED** [review]
Need fix

**Review by Palych18 — APPROVED** [review]
Done

**Pull request merged**